src/index.js

```diff
@@ -0,0 +1,146 @@
+const state = {
+    city: 'Honolulu',
+    lat: 21.3099,
+    long: 157.8581,
+    temp: 72,
+};
+
+const convertKtoF = (temp) => {
+    return (temp - 273.15) * (9 / 5) + 32;
+};
+
+const findLatitudeAndLongitude = () => {
+    let location = state.city;
+    axios.get('http://localhost:5000/location',
```

Comment:
Prefer to reference the URL with a variable instead of string literal, maybe something like: 
`const LOCATION_URL = http://localhost:5000/location;`